src/app/components/edit-user-card/edit-user-card.component.ts

```diff
@@ -105,6 +113,17 @@ export class EditUserCardComponent implements OnInit {
     }
   }
 
+  onImageMissing(image: HTMLImageElement): void {
+    image.src = this.fallbackImageUrl;
+  }
+
+  isCardInvalid(): boolean {
+    return !(
+      this.userForm.valid &&
+      !this.image?.nativeElement.src.endsWith(this.fallbackImageUrl)
```

Comment:
We could add an additional form input, that just holds the image url, then we don't need the direct HTML element ref and can mark it as required.